app/models/host.rb

```diff
@@ -1,16 +1,10 @@
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "util/xml")
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "util/win32")
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "metadata/linux")
```

Comment:
I was concerned with the metadata code and the related code paths that dont' really have much test coverage.